R/utils-fs.R

```diff
@@ -50,6 +50,15 @@ file_copy_to <- function(pkg,
   file_copy(from_paths[!eq], to_paths[!eq], overwrite = overwrite)
 }
 
+# Checks init_site() first.
+create_subdir <- function(dest_base, subdir, pkg) {
```

Comment:
Can you drop the `dest_base` argument and just use `pkg$dst_path`? Then I think it would make the most sense to have `pkg` as the first argument.

Reply:
Looks cleaner indeed!